packages/studio-ui/src/utils/loadUserAssets.tsx

```diff
@@ -57,22 +58,24 @@ export function loadStyling(): void {
     ...layouts,
     ...componentMetadatas,
   ]);
-  userStyleFilepaths.forEach((filepath) => {
-    void dynamicImportFromBrowser(filepath).then((styling) => {
-      const styleEl = document.createElement("style");
-      styleEl.innerText = styling.default;
-      styleEl.setAttribute(USER_CUSTOM_STYLE_ATTRIBUTE, filepath);
-      document.head.appendChild(styleEl);
-      styleEl.disabled = true;
-    });
+  return userStyleFilepaths.map((filepath) => {
+    return dynamicImportFromBrowser(filepath)
+      .then((styling) => {
+        const styleEl = document.createElement("style");
+        styleEl.innerText = styling.default;
+        styleEl.setAttribute(USER_CUSTOM_STYLE_ATTRIBUTE, filepath);
+        document.head.appendChild(styleEl);
```

Comment:
Now that we have a slice in store for the loaded styles, do we still need to add these style tags to the parent document head? Or could we do the dynamic import and add the styling in the slice so `useInjectUserStyles` can just construct the tags from there?

Reply:
yes! We can do that in a future PR